### ISSUE TITLE
Do you trade with this?

### ISSUE THREAD

**Issue by botzill** [ssssi/freqtrade_strs#8]
Hi.

Thanks for sharing, from the code I see some solid strategies.

Do you currently trade with any of this? I'm new to this and want to try some.

thanks.

**Comment by ssssi** [comment]
> Hi.
> 
> Thanks for sharing, from the code I see some solid strategies.
> 
> Do you currently trade with any of this? I'm new to this and want to try some.
> 
> thanks.

yes.but i I have been optimizing, but I have not found the best optimized parameter so far，my repo all str can optimized，you can hyper by youself and find the best params

**Comment by ssssi** [comment]
this is my latest dry run:
ROI: Closed trades
∙ 83.924 USDT (2.31%) (8.39 Σ%)
∙ 563.127 CNY
ROI: All trades
∙ 80.604 USDT (1.75%) (8.06 Σ%)
∙ 540.851 CNY
Total Trade Count: 14
First Trade opened: 20 hours ago
Latest Trade opened: 2 minutes ago
Win / Loss: 10 / 1
Avg. Duration: 2:48:22
Best Performing: MATIC/USDT: 4.67%
Trading volume: 8523.446 USDT
Profit factor: 3.11
Max Drawdown: 3.72% (39.863 USDT)
